src/index.ts

```diff
@@ -82,6 +83,23 @@ async function transform(destPath: string): Promise<void> {
     }
 }
 
+// Minify file with terser.
+async function minify(destPath: string): Promise<void> {
+    try {
+        const source = await fs.readFile(destPath, 'utf8');
+
+        const result = Terser.minify(source, {});
```

Comment:
Tried this locally and it looks like `compress` and `mangle` are `true` by default, so that's good.

Can you also add `module: true` here? I had really good results when using the cli `--module` option yesterday.

> `module`: Use when minifying an ES6 module. "use strict" is implied and names can be mangled on the top scope

The only concern I have right now is that some users may run into minification issues if they have a src file that is not an esmodule (because we copy and run on all src files by default). That can be fixed some day in the future, not worried about this right now.